modules/cnf-image-based-upgrade-prep.adoc

```diff
@@ -0,0 +1,433 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
Why is this top level section here? Suggest removing it and moving the `Creating ConfigMap objects for the image-based upgrade with {lcao}` title up.